doc/api/http2.md

```diff
@@ -2055,7 +2055,9 @@ changes:
     queued to be sent, and unacknowledged `PING` and `SETTINGS` frames are all
     counted towards the current limit. **Default:** `10`.
   * `maxHeaderListPairs` {number} Sets the maximum number of header entries.
-    The minimum value is `4`. **Default:** `128`.
+    This is similar to `http.Server#maxHeadersCount` or
```

Comment:
I think this and the other ones should be links (as described by @Himself65) so that users don't have to figure out where to find the documentation, especially since the documentation uses `server.maxHeadersCount` rather than `http.Server#maxHeadersCount` so people unfamiliar with the latter notation might not know if its the right thing or not. Having it link there makes it clear that it's the right thing.

Reply:
Thank you for pointing this out. I've made these changes. :)